File: ros_ws/src/my_robot_tutorials/scripts/robot_news_station.py
#!/usr/bin/env python

import rospy
from std_msgs.msg import String


def main(args=None):
    rospy.init_node("robot_news_station")

    pub = rospy.Publisher("robot_news_radio", String, queue_size=10)

    rate = rospy.Rate(2)

    while not rospy.is_shutdown():
        msg = String()
        msg.data = "Hello there"
        pub.publish(msg)
        rate.sleep()

    rospy.loginfo("Node has been killed")


if __name__ == "__main__":
    main()

File: ros_ws/src/my_robot_tutorials/src/robot_news_station.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char **argv)
{
    ros::init(argc, argv, "robot_news_radio", ros::init_options::AnonymousName);
    ros::NodeHandle nh;
    ros::Publisher publisher = nh.advertise<std_msgs::String>("robot_news_radio", 10);
    ROS_INFO("robot_news_station node has been created");

    ros::Rate rate(2);

    while (ros::ok()) {
        std_msgs::String msg;
        msg.data = "Hello there";
        publisher.publish(msg);
        rate.sleep();
    }

    return 0;
}